<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>1839</x>
      <y>40</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnQuit{	
	background-color: transparent;
	color: rgb(0, 0, 0);
	border-radius:5px;
}
QPushButton#btnQuit:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(255, 65, 65), stop:1  rgb(255, 65, 65));
}
QPushButton#btnQuit:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(255, 65, 65);
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/icons/feather/x.svg</normaloff>:/icons/feather/x.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="bg1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>4</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>650</y>
      <width>641</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Öğretim Görevlileri&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddLessonPage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnAddLessonPage{	
	background-color: rgb(74, 149, 223);
	color: white;
	border-radius:25px;
}
QPushButton#btnAddLessonPage:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(74, 149, 223), stop:1  rgb(74, 149, 223));
}
QPushButton#btnAddLessonPage:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(74, 149, 223);
}</string>
    </property>
    <property name="text">
     <string>Ders Ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddUserPage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>291</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnAddUserPage{	
	background-color: rgb(74, 149, 223);
	color: white;
	border-radius:25px;
}
QPushButton#btnAddUserPage:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(74, 149, 223), stop:1  rgb(74, 149, 223));
}
QPushButton#btnAddUserPage:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(74, 149, 223);
}</string>
    </property>
    <property name="text">
     <string>Akademisyen Kayıt Ekle</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="listUsers">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>750</y>
      <width>1321</width>
      <height>301</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:18pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>320</y>
      <width>431</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Toplam Öğrenci Sayısı&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddStudent">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>291</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btnAddStudent{	
	background-color: rgb(74, 149, 223);
	color: white;
	border-radius:25px;
}
QPushButton#btnAddStudent:hover{	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgb(74, 149, 223), stop:1  rgb(74, 149, 223));
}
QPushButton#btnAddStudent:pressed{
	padding-left:2px;
	padding-top:2px;	
	background-color: rgb(74, 149, 223);
}</string>
    </property>
    <property name="text">
     <string>Öğrenci Kayıt Ekle</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4_3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>320</y>
      <width>431</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Toplam Ders Sayısı&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4_4">
    <property name="geometry">
     <rect>
      <x>1390</x>
      <y>320</y>
      <width>461</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Toplam Öğretmen Sayısı&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title4_5">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>420</y>
      <width>431</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title4_6">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>420</y>
      <width>431</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title4_7">
    <property name="geometry">
     <rect>
      <x>1400</x>
      <y>420</y>
      <width>431</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
